lib/hashgraph.cc

```diff
@@ -276,29 +276,22 @@ void Hashgraph::consume_sequence_and_tag(const std::string& seq,
 //
 
 // TODO? Inline in header.
-void
-Hashgraph::
-consume_fasta_and_tag(
-    std:: string const  &filename,
-    unsigned int	      &total_reads, unsigned long long	&n_consumed
+template<typename SeqIO>
```

Comment:
Ok.

I have no speed data on `consume_seq<SomeReader>(fname)` vs `consume(fname, SomeReaderInstance)` vs passing in the reader when constructing the Hashtable and storing it as a member.